docs_user/modules/proc_migrating-existing-daemons-to-target-nodes.adoc

```diff
@@ -140,15 +131,11 @@ mgr  advanced  mgr/dashboard/ALERTMANAGER_API_HOST   http://172.17.3.83:9093
 mgr  advanced  mgr/dashboard/PROMETHEUS_API_HOST     http://172.17.3.83:9092
 mgr  advanced  mgr/dashboard/GRAFANA_API_URL         https://172.17.3.144:3100
 ----
-
-. The Ceph Dashboard (mgr module plugin) has not been impacted at all by this
-relocation. The service is provided by the Ceph Manager daemon, hence we might
-experience an impact when the active mgr is migrated or is force-failed.
-However, having three replicas definition allows to redirect requests to a
-different instance (it’s still an A/P model), hence the impact should be
++
+[NOTE]
+The Ceph Dashboard (mgr module plugin) is not impacted by the relocation. The service is provided by the Ceph Manager daemon. You might experience an impact when the active `mgr` daemon is migrated or is force-failed. However, defining 3 replicas allows you to redirect requests to a different instance (it’s still an A/P model), hence the impact should be
```

Comment:
(mgr module plugin) - I’m unfamiliar with this term. Why is it here in parentheses?

Reply:
The Ceph Dahboard is made by two parts:
- external monitoring stack
- a ceph Manager plugin 

The latter is what customers use to access the service, and as per this guide, is not migrated until we migrate the Ceph Manger daemons. For this reason I thought it was useful to mention that while the monitoring stack is migrated, the Ceph Dashboard is not until the ceph-mgr is migrated to target nodes (later in the procedure).
We can remove "mgr module plugin" from parentheses, and specify that the subject of that small sentence is the Ceph dashboard as the service provided by the Ceph mgr.

```
The Ceph Dashboard is a web-based Ceph management-and-monitoring tool that can be used to inspect and administer resources in the cluster. It is implemented as a [Ceph Manager Daemon](https://docs.ceph.com/en/latest/mgr/#ceph-manager-daemon) module.
```

Reply:
@fmount The rewrite looks like the following:
"The Ceph Dashboard, as the service provided by the Ceph `mgr`, is not impacted by the relocation. You might experience an impact when the active `mgr` daemon is migrated or is force-failed. However, you can define 3 replicas to redirect requests to a different instance (it’s still an A/P model), so that the impact is limited." 

One follow-up question: What is an A/P model?

Reply:
That sounds ok to me. I would just change it from:  "However, you can define 3 replicas to redirect requests to a different instance" to "However, you can define 3 replicas in the Ceph Manager configuration to redirect requests to a different instance".
How does it sound?

A/P means "Active/Passive", and I'm ok to remove it (too much internal details that might just cause additional questions).